src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
```

Comment:
it's better to give more explainable naming
```suggestion
export enum FilterStatus {
```